.gitignore

```diff
@@ -380,6 +380,7 @@ Version.cpp
 /pwiz_tools/Bumbershoot/**/AssemblyInfo.cs
 /pwiz_tools/Bumbershoot/idpicker/Resources/Resources.*
 /pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/Properties/AssemblyInfo.cs
+/pwiz_tools/Skyline/Model/Koina/Config/KoinaConfig.xml
```

Comment:
This is being fixed in PR#3004 by changing the Koina/Config/.gitignore which was referencing only PrositConfig files.